server_admin/topics/user-federation/ldap.adoc

```diff
@@ -1,197 +1,152 @@
-      in a way that a particular user attribute is preferably read from the database instead of from the LDAP. However when you later change the edit mode,
-      the mappers configuration will not be changed as it is not easily possible to detect if they were manually changed in the meantime.
-      This means that it is recommended NOT to update the `Edit Mode` switch, but rather always decide on `Edit Mode` when creating the
-      LDAP provider. This applies for `Import Users` switch as well.
+[NOTE]
+====
+When {project_name} creates the LDAP provider, {project_name} also creates a set of initial <<_ldap_mappers,LDAP mappers>>. {project_name} configures these mappers based on a combination of the `Vendor`, `Edit Mode`, and `Import Users` switches. For example, when edit mode is UNSYNCED, {project_name} configures the mappers to read a particular user attribute from the database and not from the LDAP server. However, if you later change the edit mode, the mapper's configuration does not change because it is impossible to detect if the configuration changes changed in UNSYNCED mode. Decide the `Edit Mode` when creating the LDAP provider. This note applies to `Import Users` switch also.
+====
 
-==== Other config options
+==== Other configuration options
 
 Console Display Name::
-  Name used when this provider is referenced in the admin console
+The name of the provider to display in the admin console.
 
 Priority::
-  The priority of this provider when looking up users or adding a user.
+The priority of the provider when looking up users or adding a user.
 
 Sync Registrations::
-  Does your LDAP support adding new users?  Click this switch if you want new users created by {project_name} in the admin console or the registration page
-  to be added to LDAP.
+Toggle this switch to *ON* if you want new users created by {project_name} added to LDAP.
 
 Allow Kerberos authentication::
-  Enable Kerberos/SPNEGO authentication in realm with users data provisioned from LDAP.
-  More info in <<_kerberos,Kerberos section>>.
+Enable Kerberos/SPNEGO authentication in the realm with user data provisioned from LDAP. For more information, see the <<_kerberos,Kerberos section>>.
 
 Other options::
-  The rest of the configuration options should be self explanatory.
-  You can hover the mouse pointer over the tooltips in the Admin Console to see some more details about them.
+Hover the mouse pointer over the tooltips in the Admin Console to see more details about these options.
+
+==== Connecting to LDAP over SSL
 
-==== Connect to LDAP over SSL
+When you configure a secure connection URL to your LDAP store (for example,`ldaps://myhost.com:636'), {project_name} uses SSL to communicate with the LDAP server. You must configure a truststore on the {project_name} server side. If you do not, {project_name} cannot trust the SSL connection to LDAP.
```

Comment:
Drop the "You must"
Configure a truststore on the {project_name} server side so that {project_name} can  trust the SSL connection to LDAP.